### PR TITLE
Add TinyMCEEditorField

### PR DESCRIPTION
**Pull request by sinisaos**
Related to #130

### CODE REVIEW

File: starlette_admin/templates/forms/tinymce.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        class="form-control {%if error%}{{field.error_class}}{%endif%}" {{ field.input_params()
+        }}>{%if data%}{{data}}{%endif%}</textarea>
+    {% if field.help_text %}
+    <small class="form-hint">{{field.help_text}}</small>
+    {%endif%}
+</div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tinymce/6.4.0/tinymce.min.js"
+    integrity="sha512-nv2Ftve23IDZqQhji5P2w17Ch88OR37z6tV6djv8U6hcjpRjDXRypN6sXkN6UQo8S+/qf67LVh1a3COduJIG3Q=="
+    crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+<script>
```
Comment (jowilf) [inline starlette_admin/templates/forms/tinymce.html:12]: Remove from the template and override this -> https://jowilf.github.io/starlette-admin/api/fields/#starlette_admin.fields.BaseField.additional_js_links

File: starlette_admin/fields.py [hunk 2]
```diff
@@ -244,6 +255,15 @@ def input_params(self) -> str:
         )
 
 
+@dataclass
+class TinyMCEEditorField(TextAreaField):
+    """This field is an optional field used to represent a WYSIWYG editor
+    for long text content.
+    """
+
+    form_template: str = "forms/tinymce.html"
```
Comment (jowilf) [inline starlette_admin/fields.py:264]:
```suggestion
    class_: str = "field-tinymce-editor form-control"
    form_template: str = "forms/tinymce.html"
```

File: starlette_admin/templates/forms/tinymce.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            statusbar: false,
+            toolbar: 'undo redo | formatselect | ' +
+                'bold italic backcolor | alignleft aligncenter ' +
+                'alignright alignjustify | bullist numlist outdent indent | ' +
+                'removeformat',
+            content_style: 'body { font-family: -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif; font-size: 14px; -webkit-font-smoothing: antialiased; }'
+        }
+        tinyMCE.init(options);
+    })
+</script>
```
Comment (jowilf) [inline starlette_admin/templates/forms/tinymce.html:27]: Put this inside statics/forms.js and use jquery. Something like $(".field-tinymce-editor")

**Review by jowilf — CHANGES_REQUESTED** [review]
Thank you for your work, Please check my comment

File: starlette_admin/templates/displays/text.html
Comment (jowilf) [inline starlette_admin/templates/displays/text.html:1]: It's better to create a new template for displaying  tinymceeditor data

Comment (sinisaos) [reply starlette_admin/templates/displays/text.html:1]: I will change that.

File: starlette_admin/templates/forms/tinymce.html (hunk 1 above)
Comment (sinisaos) [reply starlette_admin/templates/forms/tinymce.html:12]: I will change that.

File: starlette_admin/templates/forms/tinymce.html (hunk 3 above)
Comment (sinisaos) [reply starlette_admin/templates/forms/tinymce.html:27]: The problem with jQuery is that I haven't found a single download that works or a cdn without an api key. There is a TinyCloud cdn without an api key, but then a pop up in TinyMCE in form page appears and that's not right. Do you agree that I use pure JS in `form.js` and add something like this
```javascript
// at the end of form.js
let options = {
  selector: '#tinymce-editor',
  height: 300,
  menubar: false,
  statusbar: false,
  toolbar: 'undo redo | formatselect | ' +
    'bold italic backcolor | alignleft aligncenter ' +
    'alignright alignjustify | bullist numlist outdent indent | ' +
    'removeformat',
  content_style: 'body { font-family: -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif; font-size: 14px; -webkit-font-smoothing: antialiased; }'
}
tinyMCE.init(options);
```

**Comment by jowilf** [comment]
Thank you for your contribution, I will add a couple of things and merge it

**Review by jowilf — APPROVED** [review]

**Pull request merged**